### PR TITLE
Cross-post workshop report from HPC Carpentry blog

### PR DESCRIPTION
**Pull request by tobyhodges**
FYI @reid-a @tkphd this is currently scheduled to go out Thursday

### CODE REVIEW

**Comment by reid-a** [comment]
Looks good to me. Fire when ready.

File: _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+layout: page
+authors: ["Andrew Reid", Trevor Keller", "Jane Herriman"]
+teaser: "Report from a recent HPC Carpentry workshop."
```
Comment (OscarSiba) [inline _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md]:
```suggestion
teaser: "Report from a recent HPC Carpentry workshop at Lawrence Livermore National Laboratory."
```

File: _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+date: 2024-08-13
+time: "12:00:00"
+tags: ["HPC Carpentry", "Lesson Program Implementation"]
+---
+
+_This is a cross-posting from [the HPC Carpentry blog][original], reporting on a recent workshop._
+
+## HPC Carpentry at LLNL
+
+In the first week of June, 2024, instructors from [HPC Carpentry][hpcc]
```
Comment (OscarSiba) [inline _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md]:
```suggestion
In the first week of June, 2024, Instructors from [HPC Carpentry][hpcc]
```

File: _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Workshop Structure
+
+Each workshop ran over two days. On the first day, we did the [Unix Shell
+intro][shell] lesson from Software Carpentry in the morning, and our own
+[HPC Intro][intro] lesson in the afternoon. On the second day, we did a
+variant of the [workflow lesson][work], adapted for the Maestro workflow
+tool (rather than Snakemake), because it is developed and used at LLNL.
+
+The instructor team consisted of Andrew Reid and Trevor Keller from
```
Comment (OscarSiba) [inline _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md]:
```suggestion
The Instructor team consisted of Andrew Reid and Trevor Keller from
```

File: _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+the HPC Carpentry steering committee, and Jane Herriman from LLNL,
+along with helpers from the LLNL community.
+
+While split-terminal tools exist, we used vanilla [tmux][tmux] with two
+terminals attached to the same session. This allowed the instructors to type on
```
Comment (OscarSiba) [inline _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md]:
```suggestion
terminals attached to the same session. This allowed the Instructors to type on
```

File: _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+### Learners
+
+Learners had a range of backgrounds, from undergraduate bio-informatics
+students to experienced Linux HPC users. The lessons generally went
+at a slightly faster pace than expected, without leaving anyone
+behind. This was in part because access to LLNL's system `Ruby` was by means
+of pre-authorized RSA tokens, removing a lot of the friction
+from the initial connection process that has been time-consuming in other
+versions of the workshop. The instructors live-coded plenty of mistakes, opening
```
Comment (OscarSiba) [inline _posts/2024/08/2024-08-22-hpc-carpentry-llnl-workshop-report.md]:
```suggestion
versions of the workshop. The Instructors live-coded plenty of mistakes, opening
```

**Review by OscarSiba — APPROVED** [review]
Capitalised "Instructors" as per Carpentries style guide, and expanded the teaser sentence.

**Comment by tobyhodges** [comment]
Thanks @OscarSiba this looks good to me

**Pull request merged**